<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuMain</class>
 <widget class="QMainWindow" name="MenuMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="user">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenido(a):</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="ventas">
     <attribute name="title">
      <string>Ventas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Registrar Venta</string>
      </property>
      <widget class="QLabel" name="cb_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>0</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Codigo de Barras</string>
       </property>
      </widget>
      <widget class="QLabel" name="cantidad_label">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>0</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Cantidad</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="descripcion">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>631</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>310</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="total_label">
            <property name="text">
             <string>Total Bruto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="iva_label">
            <property name="text">
             <string>IVA:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="iva">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>340</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="codigo_barras"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="cantidad">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cargar">
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;O presiona Enter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Agregar</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>360</y>
         <width>96</width>
         <height>27</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;O presiona Espacio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cobrar</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
      <widget class="QLabel" name="notify">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>361</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: green;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="totalneto">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>310</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="totalneto_label">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>320</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>360</y>
         <width>90</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="entradas">
     <attribute name="title">
      <string>Entradas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>601</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Registrar Entrada de efectivo</string>
      </property>
      <widget class="QPushButton" name="entrada_guardar">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>250</y>
         <width>90</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_entrada_cantidad">
          <property name="text">
           <string>Cantidad: $</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="entrada_cantidad"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>291</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_entrada_dscripcion">
          <property name="text">
           <string>Descripción</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="entrada_dscripcion"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="salidas">
     <attribute name="title">
      <string>Salidas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>601</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Registrar Salida de efectivo</string>
      </property>
      <widget class="QPushButton" name="salida_guardar">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>90</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_salida_cantidad">
          <property name="text">
           <string>Cantidad: $</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="salida_cantidad"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>291</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_salida_dscripcion">
          <property name="text">
           <string>Descripción, Proveedor o factura</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="salida_dscripcion"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="reporte">
     <attribute name="title">
      <string>Reporte de Movimientos</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>631</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Reporte de Movimientos</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>191</x>
         <y>0</y>
         <width>431</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Generar Reporte</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="reporte_movimientos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>611</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>290</y>
         <width>198</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Total Ingresos:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Total Egresos:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Total Ventas:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Total:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="total_ingresos">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_egresos">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_ventas">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="reporte_ventas">
     <attribute name="title">
      <string>Reporte de Ventas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>631</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Reporte de Ventas</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>61</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>F. Inicio</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>F. Fin</string>
       </property>
      </widget>
      <widget class="QTableView" name="ventas_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>611</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="masterDetail">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>611</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="button_ventas">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>0</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Generar Reporte</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="fecha_fin">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>0</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="fecha_inicio">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>0</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="total_1">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>360</y>
         <width>93</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="fecha_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>346</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QCommandLinkButton" name="button_inventario">
       <property name="text">
        <string>Inventario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="button_usuarios">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Gestionar Usuarios</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCerrar_Sesi_n"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="actionTodos_los_productos"/>
    <addaction name="actionCrear_Producto"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAyuda"/>
   </widget>
   <widget class="QMenu" name="menuUsuarios">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Usuarios</string>
    </property>
    <addaction name="actionUsuarios"/>
    <addaction name="actionCrear_Usuario"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuInventario"/>
   <addaction name="menuUsuarios"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCerrar_Sesi_n">
   <property name="text">
    <string>Cerrar Sesión</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionTodos_los_productos">
   <property name="text">
    <string>Todos los productos</string>
   </property>
  </action>
  <action name="actionCrear_Producto">
   <property name="text">
    <string>Crear Producto</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Todos los Usuarios</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCrear_Usuario">
   <property name="text">
    <string>Crear Usuario</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>codigo_barras</tabstop>
  <tabstop>cantidad</tabstop>
  <tabstop>cargar</tabstop>
  <tabstop>descripcion</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>entrada_cantidad</tabstop>
  <tabstop>entrada_dscripcion</tabstop>
  <tabstop>entrada_guardar</tabstop>
  <tabstop>salida_cantidad</tabstop>
  <tabstop>salida_dscripcion</tabstop>
  <tabstop>salida_guardar</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>button_usuarios</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
